### DISCUSSION TITLE
Can I relock the bootloader after Duo-DE is installed?

### DISCUSSION THREAD

**Discussion by voluptuary** [Archfx/duo-de#77] [Q&A] [answered]
Or will that cause issues?

**Comment by Archfx** [comment, marked as answer]
It's generally **not recommended to relock the bootloader** after installing a GSI ROM, as this can lead to a risk of bricking your device. If you have a specific reason for wanting to relock it by taking the risk, keep in mind that you should stay away from OTA updates and Magisk, which have the potential to corrupt the operating system based on your specific configuration.

**Reply by voluptuary** [reply]
Ok, good to know. I just wanted to get rid of the warning when the device boots. Thanks for the quick response.

**Comment by oilcan-productions** [comment]
I tried re-locking with the latest ROM to be able to set up our workplace profile, and it bricked the device. Had to recover it the hard way back to 12L. Required switching to the other slot, then getting it to boot into recovery mode and flash a recovery image downloaded from Microsoft.

**Reply by voluptuary** [reply]
Well thank you for testing. It's unfortunate we can't put a stable custom build on and then lock things back down without bricking the device. I suppose it's even more a shame that giant corporations like Microsoft can't just do what would be infinitely easier for them, and release updated factory ROMs. But thankfully we have these awesome independent devs doing it the hard way for us.

**Comment by oilcan-productions** [comment]
I just found, while wanting to document and debug what is not working when enabling work profile, that I was able to enable work profile without re-locking the boot loader on the latest build.
